python/tests/proton_tests/test_message.py

```diff
@@ -277,7 +275,7 @@ def testGroupSequenceEncodeAsNull(self):
         # Skip past the headers
         consumed = decoder.decode(data)
         decoder.clear()
-        data = data[consumed:]
+        # data = data[consumed:]
```

Comment:
@SudhanAruna Why did you comment this out?

Reply:
@jiridanek sry, I commented for testing purpose. I'll amend it